features/facebookImport/src/model/entities/entities-group.js

```diff
@@ -0,0 +1,11 @@
+import { fromJsonObject, toJsonObject } from "./utils/json-serialization";
+
+export default class EntitiesGroup {
+    toJSON() {
```

Comment:
If we are going to use this always, possibly an abstract base class would convey that meaning. Also, [consider `bubblewrap`](https://github.com/polypoly-eu/polyPod/tree/main/core/communication/bubblewrap)